game/templates/game/board.html

```
@@ -660,6 +632,74 @@
 
+    <!-- ========== THEME & SETTINGS MODAL ========== -->
+    <div class="theme-settings-modal" id="themeSettingsModal" role="dialog" aria-modal="true" aria-labelledby="themeModalTitle" style="display: none;">
+        <div class="theme-modal-dialog">
+            <button type="button" class="modal-close-btn" id="closeThemeModalBtn" aria-label="Close modal">&times;</button>
+            <h2 class="theme-modal-title" id="themeModalTitle">⚙️ Theme & Settings</h2>
+            
+            <div class="theme-modal-section">
+                <h3 class="section-title">Board Theme</h3>
+                <div class="theme-swatches-grid">
+                    <label class="theme-swatch-label">
+                        <input type="radio" name="boardThemeRadio" value="classic" class="theme-radio-input">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)